visualization/app/codeCharta/ui/codeMap/rendering/codeMapMesh.ts

```diff
@@ -47,6 +47,7 @@ export class CodeMapMesh {
 
 	selectBuilding(building: CodeMapBuilding, color: string) {
 		building.setColor(color)
+		building.setDeltaColor(color)
```

Comment:
I think this could introduce a hidden side effect because the delta color is never resetted (see `clearSelection()`).

Reply:
ClearSelection() calls resetColors() ([See here](https://github.com/MaibornWolff/codecharta/blob/b0b663699f156bf7f538b58958351f9097a79439/visualization/app/codeCharta/ui/codeMap/rendering/codeMapBuilding.ts#L71)) and there color and delta color are resett.